dev/io.openliberty.checkpoint_fat_mail/publish/servers/imapTestServer/server.xml

```diff
@@ -0,0 +1,28 @@
+<server>
+  <featureManager>
+    <feature>componentTest-1.0</feature>
+    <feature>servlet-4.0</feature>
+    <feature>javaMail-1.6</feature>
+  </featureManager>
+
+  <mailSession>
+    <mailSessionID>testIMAPMailSession</mailSessionID>
+    <jndiName>IMAPTestingApp/IMAPMailSessionServlet/testIMAPMailSession</jndiName>
+    <description>mailSession for testing IMAP protocol</description>
+    <storeProtocol>imap</storeProtocol>
+    <host>localhost</host>
+    <user>imap@testserver.com</user>
+    <password>${imap_password}</password>
+    <from>smtp@testserver.com</from>
+    <property name="mail.imap.host" value="localhost" />
+    <property name="mail.imap.port" value="${bvt.prop.imap_port}" />
+  </mailSession>
+
+  <include location="../fatTestPorts.xml"/>
+
+  <application id="IMAPTestingApp" name="IMAPTestingApp" type="war" location="IMAPTestingApp.war">
+    <classloader apiTypeVisibility="+third-party"/>
+  </application>
+  
+  <variable name="imap_password" defaultValue="wrong_password" />
```

Comment:
Do you need to have this default defined?  What if we have nothing set and then it gets set with the env config?

Reply:
done!